warrior/Actions/CommonActions/common_actions.py

```diff
@@ -255,3 +255,27 @@ def set_env_var(self, var_key=None, var_value=None, filepath=None,
                 status = False
 
         return status
+
+    def verify_arith_exp(self, expression, expected, comparison='eq'):
+        """ Verify the output of the arithmetic expression matches the expected(float comparison)
+            :Arguments:
+                1. expression: Arithmetic expression to be compared with expected.
+                    This can have env & data_repo values embedded in it.
+                        Ex. expression: "10+${ENV.x}-${REPO.y}*10"
+                    Expression will be evaluated based on python operator precedence
+                2. expected: Value to be compared with the expression output
```

Comment:
Just as in data_utils file, please explain the usage of expected : "This can be a env or data_repo value or any numeral value"

Reply:
@brettleehari Added, Thanks!!